--- icc-examin-0.56/configure.orig	2016-12-09 23:50:43.000000000 +0100
+++ icc-examin-0.56/configure	2023-08-28 06:26:29.924434276 +0200
@@ -346,6 +346,11 @@ else
       test -n $ECHO && $ECHO $e "$echo_" >> $CONF_LOG; test -n "$INFOECHO" && $INFOECHO "$echo_"
         DISTNAME="$echo_"
       SYS=1; fi
+    if [ -f /etc/pld-release ]; then
+      echo_="`cat /etc/pld-release`"
+      test -n $ECHO && $ECHO $e "$echo_" >> $CONF_LOG; test -n "$INFOECHO" && $INFOECHO "$echo_"
+        DISTNAME="$echo_"
+      SYS=1; fi
     if [ -f /etc/debian_version ]; then
       if [ -f /etc/issue.net ]; then
         echo_="`cat /etc/issue.net || cat /etc/debian_version`"
@@ -353,11 +358,8 @@ else
         DISTNAME="$echo_"
         SYS=2; fi; fi
     if [ $SYS -eq 0 ]; then
-      if [ -f /etc/issue.net ]; then
-        echo_="`cat /etc/issue.net`"
-        test -n $ECHO && $ECHO $e "$echo_" >> $CONF_LOG; test -n "$INFOECHO" && $INFOECHO "$echo_"
-        DISTNAME="$echo_"
-    fi; fi
+        DISTNAME="Linux"
+    fi
     OSUNAME=Linux
   else
     if [ $UNAME_ = "SunOS" ]; then
