--- icc_examin-0.55/icc_gl.cpp.orig	2014-03-16 18:28:36.436024702 +0100
+++ icc_examin-0.55/icc_gl.cpp	2014-03-16 18:31:13.036018129 +0100
@@ -865,7 +865,7 @@
   if(font) delete font;
   if(ortho_font) delete ortho_font;
 
-  char * font_name = strdup("/usr/share/fonts/dejavu/DejaVuSans.ttf");
+  char * font_name = strdup("/usr/share/fonts/TTF/DejaVuSans.ttf");
 
 # ifdef HAVE_FONTCONFIG
   FcResult r = FcResultMatch;
@@ -909,7 +909,7 @@
 # endif
   if(!holeDateiModifikationsZeit(font_name)) {
     DBG_PROG_S( _("Could not open font in:") << font_name )
-    font_name = strdup("/usr/share/fonts/truetype/DejaVuSans.ttf");
+    font_name = strdup("/usr/share/fonts/TTF/DejaVuSans.ttf");
     if(!holeDateiModifikationsZeit(font_name)) {
       DBG_PROG_S( _("Could not open font in:") << font_name )
 #ifdef  WIN32
