--- icc_examin-0.54/configure.orig	2013-01-11 12:55:26.676880980 +0100
+++ icc_examin-0.54/configure	2013-03-31 21:11:35.026149419 +0200
@@ -345,6 +345,11 @@
       test -n $ECHO && $ECHO $e "$echo_" >> $CONF_LOG; test -n "$INFOECHO" && $INFOECHO "$echo_"
         DISTNAME="$echo_"
       SYS=1; fi
+    if [ -f /etc/pld-release ]; then
+      echo_="`cat /etc/pld-release`"
+      test -n $ECHO && $ECHO $e "$echo_" >> $CONF_LOG; test -n "$INFOECHO" && $INFOECHO "$echo_"
+        DISTNAME="$echo_"
+      SYS=1; fi
     if [ -f /etc/debian_version ]; then
       if [ -f /etc/issue.net ]; then
         echo_="`cat /etc/issue.net || cat /etc/debian_version`"
@@ -352,10 +357,7 @@
         DISTNAME="$echo_"
         SYS=2; fi; fi
     if [ $SYS -eq 0 ]; then
-      if [ -f /etc/issue.net ]; then
-        echo_="`cat /etc/issue.net`"
-        test -n $ECHO && $ECHO $e "$echo_" >> $CONF_LOG; test -n "$INFOECHO" && $INFOECHO "$echo_"
-        DISTNAME="$echo_"
-    fi; fi
+      DISTNAME="Linux"
+    fi
     OSUNAME=Linux
   else
