--- icc_examin-0.55/configure_tests.sh.orig	2013-05-02 13:10:35.207244645 +0200
+++ icc_examin-0.55/configure_tests.sh	2014-03-16 18:28:08.829359193 +0100
@@ -529,6 +529,7 @@
 
 if [ -n "$XCM" ] && [ $XCM -gt 0 ]; then
     found=""
+    cflags=""
     version=""
     pc_package=xcm
     name="xcm"
@@ -581,7 +582,8 @@
     if [ -z "$found" ]; then
       pkg-config  --atleast-version=1.0 $pc_package
       if [ $? = 0 ]; then
-        found=`pkg-config --cflags $pc_package`
+	found=yes
+        cflags=`pkg-config --cflags $pc_package`
         version=`pkg-config --modversion $pc_package`
       fi
     fi
@@ -604,22 +606,27 @@
 if [ -n "$X11" ] && [ $X11 -gt 0 ]; then
   if [ -n "$XF86VMODE" ] && [ $XF86VMODE -gt 0 ]; then
     found=""
+    cflags=""
     version=""
     pc_package=xf86vidmodeproto
     if [ -z "$found" ]; then
       pkg-config  --atleast-version=1.0 $pc_package
       if [ $? = 0 ]; then
-        found=`pkg-config --cflags $pc_package`
+        found=yes
+        cflags=`pkg-config --cflags $pc_package`
         version=`pkg-config --modversion $pc_package`
       fi
     fi
     if [ -z "$found" ]; then
       if [ -f /usr/X11R6/include/X11/extensions/xf86vmode.h ]; then
-        found="-I/usr/X11R6/include"
+        found=yes
+        cflags="-I/usr/X11R6/include"
       elif [ -f /usr/include/X11/extensions/xf86vmode.h ]; then
-        found="-I/usr/include"
+        found=yes
+        cflags="-I/usr/include"
       elif [ -f $includedir/X11/extensions/xf86vmode.h ]; then
-        found="-I$includedir"
+        found=yes
+        cflags="-I$includedir"
       fi
     fi
     if [ -n "$found" ]; then
@@ -632,7 +639,7 @@
       if [ -n "$MAKEFILE_DIR" ]; then
         for i in $MAKEFILE_DIR; do
           test -f "$ROOT_DIR/$i/makefile".in && echo "XF86VMODE = 1" >> "$i/makefile"
-          test -f "$ROOT_DIR/$i/makefile".in && echo "XF86VMODE_INC = $found" >> "$i/makefile"
+          test -f "$ROOT_DIR/$i/makefile".in && echo "XF86VMODE_INC = $cflags" >> "$i/makefile"
         done
       fi
     elif [ $OSUNAME = "Linux" ]; then
@@ -644,22 +651,27 @@
 
   if [ -n "$XINERAMA" ] && [ $XINERAMA -gt 0 ]; then
     found=""
+    cflags=""
     version=""
     pc_package=xinerama
     if [ -z "$found" ]; then
       pkg-config  --atleast-version=1.0 $pc_package
       if [ $? = 0 ]; then
-        found=`pkg-config --cflags $pc_package`
+        found=yes
+        cflags=`pkg-config --cflags $pc_package`
         version=`pkg-config --modversion $pc_package`
       fi
     fi
     if [ -z "$found" ]; then
       if [ -f /usr/X11R6/include/X11/extensions/Xinerama.h ]; then
-        found="-I/usr/X11R6/include"
+        found=yes
+        cflags="-I/usr/X11R6/include"
       elif [ -f /usr/include/X11/extensions/Xinerama.h ]; then
-        found="-I/usr/include"
+        found=yes
+        cflags="-I/usr/include"
       elif [ -f $includedir/X11/extensions/Xinerama.h ]; then
-        found="-I$includedir"
+        found=yes
+        cflags="-I$includedir"
       fi
     fi
     if [ -n "$found" ]; then
@@ -672,7 +684,7 @@
       if [ -n "$MAKEFILE_DIR" ]; then
         for i in $MAKEFILE_DIR; do
           test -f "$ROOT_DIR/$i/makefile".in && echo "XIN = 1" >> "$i/makefile"
-          test -f "$ROOT_DIR/$i/makefile".in && echo "XINERAMA_INC = $found" >> "$i/makefile"
+          test -f "$ROOT_DIR/$i/makefile".in && echo "XINERAMA_INC = $cflags" >> "$i/makefile"
         done
       fi
     elif [ $OSUNAME = "Linux" ]; then
@@ -684,12 +696,14 @@
 
   if [ -n "$XRANDR" ] && [ $XRANDR -gt 0 ]; then
     found=""
+    cflags=""
     version=""
     pc_package=xrandr
     if [ -z "$found" ]; then
       pkg-config  --atleast-version=1.2 $pc_package
       if [ $? = 0 ]; then
-        found=`pkg-config --cflags $pc_package`
+        found=yes
+        cflags=`pkg-config --cflags $pc_package`
         version=`pkg-config --modversion $pc_package`
       fi
     fi
@@ -703,7 +717,7 @@
       if [ -n "$MAKEFILE_DIR" ]; then
         for i in $MAKEFILE_DIR; do
           test -f "$ROOT_DIR/$i/makefile".in && echo "XRANDR = 1" >> "$i/makefile"
-          test -f "$ROOT_DIR/$i/makefile".in && echo "XRANDR_INC = $found" >> "$i/makefile"
+          test -f "$ROOT_DIR/$i/makefile".in && echo "XRANDR_INC = $cflags" >> "$i/makefile"
         done
       fi
     elif [ $OSUNAME = "Linux" ]; then
@@ -841,22 +855,27 @@
 fi
 if [ -n "$X11" ] && [ $X11 -gt 0 ]; then
     found=""
+    cflags=""
     version=""
     pc_package=x11
     if [ -z "$found" ]; then
       pkg-config  --atleast-version=1.0 $pc_package
       if [ $? = 0 ]; then
-        found=`pkg-config --cflags $pc_package`
+        found=yes
+        cflags=`pkg-config --cflags $pc_package`
         version=`pkg-config --modversion $pc_package`
       fi
     fi
     if [ -z "$found" ]; then
       if [ -f /usr/X11R6/include/X11/Xlib.h ]; then
-        found="-I/usr/X11R6/include"
+        cflags="-I/usr/X11R6/include"
+	found=yes
       elif [ -f /usr/include/X11/Xlib.h ]; then
-        found="-I/usr/include"
+        cflags="-I/usr/include"
+	found=yes
       elif [ -f $includedir/X11/Xlib.h ]; then
-        found="-I$includedir"
+        cflags="-I$includedir"
+	found=yes
       fi
     fi
     if [ -n "$found" ]; then
@@ -869,7 +888,7 @@
       if [ -n "$MAKEFILE_DIR" ]; then
         for i in $MAKEFILE_DIR; do
           test -f "$ROOT_DIR/$i/makefile".in && echo "X11 = X11" >> "$i/makefile"
-          test -f "$ROOT_DIR/$i/makefile".in && echo "X_H = -I/usr/X11R6/include -I/usr/include $found \$(X11_INCL)" >> "$i/makefile"
+          test -f "$ROOT_DIR/$i/makefile".in && echo "X_H = $cflags \$(X11_INCL)" >> "$i/makefile"
         done
       fi
     elif [ $OSUNAME = "Linux" ]; then
